index.html

```diff
@@ -133,18 +133,18 @@ <h3 class="article__title">How can I help?</h3>
                     </a>
                   </p>
                 </div>
-              </div>
+              </article>
             </div>
           </div>
         </div>
 
-        <div class="sidebar col-lg-4">
+        <aside class="sidebar col-lg-4">
           <h2 class="heading-underline">Upcoming events</h2>
 
-          <div class="article">
+          <article class="article">
             <img
```

Comment:
You can remove the `class="article"` attribute and target the `article` tag directly in the CSS (change `.article` to article). This applies to all of your `<article>` elements in this document.